src/vde_backend/alerts.py

```diff
@@ -20,7 +20,7 @@
 from datetime import timedelta
 
 # local source
-import pyToCassandra as ptc
+import cassandra_driver as driver
```

Comment:
Why renaming `ptc` to `driver`?
It can introduce new bugs, and it is less clear in my opinion. We have SFTP, RTU, and Cassandra drivers. Thus, something called `driver` is not really meaningful.